src/components/shared/TreeView.js

```diff
@@ -231,6 +229,7 @@ class TreeViewRowScrolledColumns<
           className={`treeRowToggleButton ${
             isExpanded ? 'expanded' : 'collapsed'
           } ${canBeExpanded ? 'canBeExpanded' : 'leaf'}`}
+          onClick={this.props.onToggle}
```

Comment:
when you do this, the function `onToggle` will be called with one parameter, with type `SyntheticEvent`. But as you can see in the (previous) line 195, `onToggle` needs to be called with 3 arguments. That's why you need to create a local function `onToggleClick` that will call `onToggle` properly with the arguments, using the `props` and the parameter.